docs/tutorials/wiki2/tests.rst

```diff
@@ -88,7 +88,7 @@ Update ``tests/conftest.py`` to look like the following, adding the highlighted
 
 .. literalinclude:: src/tests/tests/conftest.py
     :linenos:
-    :emphasize-lines: 10,68-103,110,117-119
+    :emphasize-lines: 68-103,110,117-119
```

Comment:
Revert this, the import wasn't supposed to be in the cookiecutter.

Reply:
Done in 6d3e713